WPMedia/fusion-news-theme-blocks
# Tbrands 110 image featured algolia

## Description

Show featured image for algolia schema change 

## Jira Ticket

- [TBRANDS-110](https://arcpublishing.atlassian.net/browse/TBRANDS-110)

## Acceptance Criteria

Show image from featured image schema 

## Test Steps

1. Checkout this branch `git checkout TBRANDS-110-image-featured-algolia`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/product-assortment-carousel-block`
3. Add commerce update per https://github.com/WPMedia/arc-themes-blocks/pull/1528
4. See the images in the featured image schema for the product 
<img width="1346" alt="Screen Shot 2022-10-18 at 08 45 21" src="https://user-images.githubusercontent.com/5950956/196447490-bfc22cef-f430-449d-ab0b-5fa3a1d8864b.png">


## Effect Of Changes

### Before

no image shown 

![image](https://user-images.githubusercontent.com/5950956/196447605-2395134f-0c8b-47d8-b926-440bd106fd2e.png)


### After

image shown 
<img width="686" alt="Screen Shot 2022-10-18 at 08 31 26" src="https://user-images.githubusercontent.com/5950956/196447567-87ae0eae-452b-44a0-a368-979f1c978c8d.png">


## Dependencies or Side Effects
- Fixed the eslintignore for storybook

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored). -> tests working as before
- [x] Confirmed relevant documentation has been updated/added. -> none additional needed

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
